scripts/zones/Aht_Urhgan_Whitegate/npcs/Waoud.lua

```diff
@@ -13,33 +13,35 @@ require("scripts/globals/npc_util")
 local ID = require("scripts/zones/Aht_Urhgan_Whitegate/IDs")
 -----------------------------------
 
-function onTrade(player,npc,trade)
-    local anEmptyVessel = player:getQuestStatus(AHT_URHGAN,tpz.quest.id.ahtUrhgan.AN_EMPTY_VESSEL)
+function onTrade(player, npc, trade)
+    local anEmptyVessel = player:getQuestStatus(AHT_URHGAN, tpz.quest.id.ahtUrhgan.AN_EMPTY_VESSEL)
     local anEmptyVesselProgress = player:getCharVar("AnEmptyVesselProgress")
     local StoneID = player:getCharVar("EmptyVesselStone")
 
     -- AN EMPTY VESSEL (dangruf stone, valkurm sunsand, or siren's tear)
-    if anEmptyVessel == QUEST_ACCEPTED and anEmptyVesselProgress == 3 and trade:hasItemQty(StoneID,1) and trade:getItemCount() == 1 then
-        player:startEvent(67,StoneID) -- get the stone to Aydeewa
+    if anEmptyVessel == QUEST_ACCEPTED and anEmptyVesselProgress == 3 and trade:hasItemQty(StoneID, 1) and trade:getItemCount() == 1 then
+        player:startEvent(67, StoneID) -- get the stone to Aydeewa
     end
 end
 
-function onTrigger(player,npc)
-    local anEmptyVessel = player:getQuestStatus(AHT_URHGAN,tpz.quest.id.ahtUrhgan.AN_EMPTY_VESSEL)
+function onTrigger(player, npc)
+    local anEmptyVessel = player:getQuestStatus(AHT_URHGAN, tpz.quest.id.ahtUrhgan.AN_EMPTY_VESSEL)
     local anEmptyVesselProgress = player:getCharVar("AnEmptyVesselProgress")
     local divinationReady = vanaDay() > player:getCharVar("LastDivinationDay")
-    local beginnings = player:getQuestStatus(AHT_URHGAN,tpz.quest.id.ahtUrhgan.BEGINNINGS)
+    local beginnings = player:getQuestStatus(AHT_URHGAN, tpz.quest.id.ahtUrhgan.BEGINNINGS)
+    local omens = player:getQuestStatus(AHT_URHGAN, tpz.quest.id.ahtUrhgan.OMENS)
+    local needToZone = player:getLocalVar("WaoudNeedToZone")
```

Comment:
there is a function player:needToZone()

Reply:
ah thanks for the catch @KnowOne134 , let me murder it with kindness

Reply:
@KnowOne134 i dont see it on my end. You might be looking at a wrong commit?

Reply:
just general you don't want a var (in this case `local needToZone`) that has same name as a function. I recall ibm wanting to phase said function out.

Reply:
alright. Changing it @TeoTwawki  😸

Reply:
changed to waoudNeedToZone :smile_cat: 

resolving comment